src/pages/RFOX/hooks/useGetUnstakingRequestQuery.ts

```diff
@@ -103,5 +105,11 @@ export const useGetUnstakingRequestQuery = <SelectData = UnstakingRequest>({
     retry: false,
   })
 
-  return unstakingRequestQuery
+  return {
+    ...unstakingRequestQuery,
+    isError: getUnstakingRequestQueryFn !== skipToken && unstakingRequestQuery.isError,
+    // Note, this isn't part of the usual react-query return type, but allows us to handle the weirdness of skipToken making the query in an error state
+    // until the *actual* queryFn invoked
+    isEnabled: getUnstakingRequestQueryFn !== skipToken,
```

Comment:
As mentioned in the comment and in discussions with eng., I'm confident this is caused by the tanstack `useQuery` import being the one from `wagmi/query` somewhere earlier down the component tree, and will most likely fixed by removing all occurences in a follow-up, since I couldn't repro in a clean codesandbox https://codesandbox.io/p/sandbox/great-snyder-t5vxyk?file=/src/index.js